app/components/modal/edit-user/template.hbs

```diff
@@ -0,0 +1,64 @@
+<ModalDialog>
+  <div class='modal-dialog modal-xl'>
+    <div class='modal-content'>
+      <div class='modal-header'>
+        <h5 class='modal-title'>Edit user: {{@user.username}}</h5>
+      </div>
+      <div class='modal-body'>
+        <div class='mb-3'>
+          <label for='emailUserInput' class='form-label'>Enter new email</label>
+          <input
+            type='e-mail'
+            class='form-control'
+            id='editEmailInput'
+            placeholder='{{@user.email}}'
+            value={{this.email}}
+            {{on 'input' this.onEmailChange}}
+          />
+        </div>
+        <div class='mb-3'>
+          <label for='passwordUserInput' class='form-label'>Enter new password</label>
+          <input
+            type='password'
+            class='form-control'
+            id='editPasswordInput'
+            placeholder='{{@user.password}}'
+            value={{this.password}}
+            {{on 'input' this.onPasswordChange}}
+          />
+        </div>
+        <div class='mb-3'>
+          <label for='userTypeInput' class='form-label'>Change user type:</label>
+          <div {{on 'input' (fn this.onUserTypeChange 'userType')}}>
```

Comment:
Linter not shout about add `role="button"`? 🤔 https://github.com/ember-template-lint/ember-template-lint/issues/41